README.md

```diff
@@ -75,6 +75,76 @@ Open a terminal and run the following commands:
 
 </details>
 
+<details>
+
+<summary>Install prerequisites for Linux/WSL2</summary>
+
+# Installing Dependencies on Linux
```

Comment:
Since this is a <summary> section it would be great if we did not use headings, as they "conflict" with the headings in the main readme.